index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+    <meta name="" content="">
+    <link rel="stylesheet" href="styles.css" title="" type="text/css" />
+    <title></title>
+  </head>
+  <body id="body">
+    <div id="div_1">
+       <img id="tool_bar" src="/Image/Tool_bar.png" alt="This is a nav-bar." />
+       </div>
+      <div id="div_2">
+        <img id="headline" src="/Image/Headline_section.png" alt="This is the Headline." />
+      </div>
```

Comment:
- [ ] kindly work in your project to match the one in the Figma not extract the whole image from design and add it.